technologies/markup.md

```diff
@@ -0,0 +1,48 @@
+# Markup
```

Comment:
This whole doc is an interesting read! Please could you add a tl;dr at the top and summarise the primary piece of information a reader should take away from this file?